### PR TITLE
WEBUI-841: create layouts for userworkspaces (Backport 10.10)

### PR DESCRIPTION
**Pull request by GuruprasadAnk1996**
https://jira.nuxeo.com/browse/WEBUI-841

### CODE REVIEW

**Review by rakeshkumar1019 — DISMISSED** [review]
looks good

File: elements/document/userworkspacesroot/nuxeo-userworkspacesroot-view-layout.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<!--
+`nuxeo-userworkspacesroot-view-layout`
+@group Nuxeo UI
+@element nuxeo-userworkspacesroot-view-layout
+-->
+<dom-module id="nuxeo-userworkspacesrot-view-layout">
```
Comment (semisse) [inline elements/document/userworkspacesroot/nuxeo-userworkspacesroot-view-layout.html]:
```suggestion
<dom-module id="nuxeo-userworkspacesroot-view-layout">
```

**Review by semisse — CHANGES_REQUESTED** [review]
Please fix the misspelled `id` I mentioned before

**Review by semisse — APPROVED** [review]

**Review by semisse — CHANGES_REQUESTED** [review]
You have two commits, please squash them into one

**Pull request closed without merging**